### PR TITLE
Created customerprofile component

### PR DESCRIPTION
**Pull request by princegeorge121**
Created a layout.

### CODE REVIEW

File: client/mycart/src/app/customer/customer-profile/customer-profile.component.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                            <div class="col-sm-3">
+                                <p class="mb-0">Address</p>
+                            </div>
+                            <div class="col-sm-9">
+                                <p class="text-muted mb-0">Bay Area, San Francisco, CA</p>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="d-flex justify-content-center align-items-center h-50">
+                        <button type="button" class="btn btn-primary w-50 ">Edit Profile</button>
```
Comment (mishelshaji) [inline client/mycart/src/app/customer/customer-profile/customer-profile.component.html:64]: Remove the extra whitespace from the end.

**Review by mishelshaji — CHANGES_REQUESTED** [review]

File: client/mycart/src/app/customer/customer-profile/customer-profile.component.html (hunk 1 above)
Comment (princegeorge121) [reply client/mycart/src/app/customer/customer-profile/customer-profile.component.html:64]: Done

**Comment by mishelshaji** [comment]
Did you forget to commit the changes?

File: client/mycart/src/app/customer/customer-routing.module.ts [hunk 2]
```diff
@@ -15,7 +16,8 @@ const routes: Routes = [
       { path: 'home', component: CustomerHomeComponent },
       { path: 'order-summary', component: OrderSummaryComponent },
       { path: 'feedback' , component: FeedbackComponent},
```
Comment (mishelshaji) [inline client/mycart/src/app/customer/customer-routing.module.ts]: Remove whitespace before the comma.

**Review by mishelshaji — CHANGES_REQUESTED** [review]

File: client/mycart/src/app/customer/customer-routing.module.ts (hunk 2 above)
Comment (princegeorge121) [reply client/mycart/src/app/customer/customer-routing.module.ts]: Done.

**Review by mishelshaji — APPROVED** [review]

**Pull request merged**